src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle() {
+    }
+
+    public Circle(double radius, Color color) {
+        this.radius = radius;
+        super.setColor(color);
+    }
```

Comment:
```suggestion
    public Circle(double radius, Color color) {
        super(color);
        this.radius = radius;
    }
```
and use it like this